packages/compiler/src/core/parser.ts

```diff
@@ -2418,14 +2418,23 @@ function createParser(code: string | SourceFile, options: ParseOptions = {}): Pa
     }
   }
 
+  /**
+   * Handles param-like documentation comment tags: `@param` and
+   * `@typeParam` (or `@template`).
+   *
+   * @see <a href="https://tsdoc.org/pages/tags/param/">@param</a>
```

Comment:
Not sure we should link to the typescript docs for this. 

I think instead we should make sure all the tags are documented and linkable in this doc https://microsoft.github.io/typespec/language-basics/documentation#tsdoc-doc-comments

Reply:
for this one you can maybe just link the docs page we can have a follow up to update the doc itself

Reply:
Reorganized/wrote the doc comments. 👍🏻